hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/HddsVolume.java

```diff
@@ -131,9 +131,11 @@ private HddsVolume(Builder b) throws IOException {
       this.volumeInfoMetrics =
           new VolumeInfoMetrics(b.getVolumeRootStr(), this);
 
-      LOG.info("Creating HddsVolume: {} of storage type : {} capacity : {}",
-          getStorageDir(), b.getStorageType(),
-              getVolumeInfo().get().getCapacity());
+      LOG.info("Creating HddsVolume: {} of storage type: {}, capacity: {}, available: {}",
+          getStorageDir(),
+          b.getStorageType(),
+          getVolumeInfo().get().getCapacity(),
+          getVolumeInfo().get().getAvailable());
```

Comment:
I would like to suggest:

1. implement `toString` in `SpaceUsageSource.Fixed` like:
```java
    @Override
    public String toString() {
      return "capacity: " + capacity +
          "used: " + used +
          "available: " + available;
    }
```
2. log usage in `HddsVolume` as:
```suggestion
      LOG.info("Creating HddsVolume: {} of storage type: {}, {}",
          getStorageDir(),
          b.getStorageType(),
          getCurrentUsage());
```

Reply:
Also, same change welcome in:

https://github.com/apache/ozone/blob/fc89ba6aef9bd01562f76ef19888a56950bc6939/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/DbVolume.java#L60-L62

Reply:
Good idea. Let me do that.